extensions/dicom-seg/src/components/SegmentationPanel.js

```diff
@@ -0,0 +1,466 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import cornerstoneTools from 'cornerstone-tools';
+import cornerstone from 'cornerstone-core';
+import moment from 'moment';
+
+import { utils } from '@ohif/core';
+import { ScrollableArea, TableList, Icon } from '@ohif/ui';
+
+import {
+  BrushColorSelector,
+  BrushRadius,
+  SegmentationItem,
+  SegmentItem,
+  SegmentationSelect,
+} from './index';
+
+import './SegmentationPanel.css';
+import SegmentationSettings from './SegmentationSettings';
+
+const { studyMetadataManager } = utils;
+
+const segmentationModule = cornerstoneTools.getModule('segmentation');
+const DEFAULT_BRUSH_RADIUS = segmentationModule.getters.radius || 10;
+
+const refreshViewport = () => {
+  cornerstone.getEnabledElements().forEach(enabledElement => {
+    cornerstone.updateImage(enabledElement.element);
+  });
+};
+
+/**
+ * SegmentationPanel component
+ *
+ * @param {Object} props
+ * @param {Array} props.studies
+ * @param {Array} props.viewports - viewportSpecificData
+ * @param {number} props.activeIndex - activeViewportIndex
+ * @returns component
+ */
+const SegmentationPanel = ({ studies, viewports, activeIndex }) => {
+  /* TODO: This technically defaults to 10 if undefined (bug?) */
+  const [brushRadius, setBrushRadius] = useState(DEFAULT_BRUSH_RADIUS);
+  const [brushColor, setBrushColor] = useState('rgba(221, 85, 85, 1)');
```

Comment:
We shouldn't hardcode this color, in the future the SEG may set the colorLUT to whatever it wants. Slap a `TODO` on it though, as we aren't using creation functionality yet.

Reply:
Done, thanks.